infrastructure-playbooks/purge-container-cluster.yml

```diff
@@ -645,6 +645,23 @@
         ansible_os_family == 'RedHat' and
         not is_atomic
 
+    - name: find any service-cid file left
+      find:
+        paths: /run
+        patterns: "ceph-*.service-cid"
```

Comment:
This won't match stuff like
  - grafana
  - prometheus
  - alermanager
  - node-exporter
  - rbd-target-{api,gw}
  - tcmu-runner